Add dashboard for blog posts

DonnieWest opened the pull request.

DonnieWest committed:
Add dashboard for blog posts

trepidity commented:
You got some fancy code in this! I like it. It's DRY.

trepidity commented:
You should be able to hit this at one of the following locations.
http://dev.treasury.love/
http://dev.treasury.love.s3-website-us-east-1.amazonaws.com/Dashboard.html

trepidity commented:
We probably better get some feedback before we release this into someone's hands :)

DonnieWest commented:
:-) it probably could use some feedback for when it actually submits. Right now that would be very confusing for anyone using it

trepidity committed:
Merge branch 'source' of github.com:LoveBank/LoveBank.github.io into PR-Dashboard

trepidity committed:
Do not track these dist files

trepidity committed:
Bootstrap theme and user Feedback

@donnie
This is a big of change from what you originally had. You have some cool ES6 stuff in here and I like it. I had to add jquery for toastr feedback. I also don't think bootstrap 3 is rid fo jquery yet....

I'm interested in pursuing less jquery and so would be open for changes to what I have proposed.

Anyway, there is now user feedback.

DonnieWest commented on app/Dashboard.html:
```diff
@@ -53,9 +144,39 @@
+          toastr.success("Blog Post " + response.statusText);
+          form.reset();
+        } else {
+          toastr.warning("There was a problem submitting the blog post : " + response.statusText);
```
We could probably change these to template strings - ``There was a problem submitting the blog post :  ${response.statusText}``

As far as JQuery, I _think_ Bootstrap requires JQuery, doesn't it? Not too much we can do to remove it as a dependency if that's the case

trepidity replied:
Not sure if I know what you mean by "template strings". Would you like to submit a commit with what you have in mind?
